.bunnyshell/templates/showcase-container-patterns/template.yaml

```diff
@@ -0,0 +1,16 @@
+# Template Documentation
+name: Showcase Container Patterns
+description: The `container-patterns-tpl` template demonstrates how to utilize init and sidecar containers in Bunnyshell. It provides practical examples to illustrate their setup and interaction within a containerized environment.
+tags:
+  - bunnyshell
+icons: [ 'typescript', 'postgres' ]
+categories:
+  - bunnyshelldesignpatterns
```

Comment:
the category is "bunnyshell design patterns". You can find it by using the category filter in Bunnyshell and check the `category` query string